.github/workflows/check-bitrise-ios-release-tags.yml

```diff
@@ -0,0 +1,62 @@
+name: Check Bitrise Firefox iOS Tags
+
+on:
+  schedule:
+    - cron: "0 * * * 1-5"  # every hour
+  workflow_dispatch:
+
+jobs:
+  check-bitrise:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+
+      - name: Install dependencies
+        run: pip install -r testrail/requirements.txt
+      
+      - name: Pull latest remote changes
+        run: git pull --rebase
```

Comment:
Is this necessary as it's doing a `checkout ` on line 14 off of default branch `main`